### PR TITLE
Sonar en 1853 - enabled coverage for sonar alalysis

### PR DESCRIPTION
**Pull request by pietrangelo**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: pom.xml [hunk 1]
```diff
@@ -105,6 +124,66 @@
...
+                <artifactId>sonar-maven-plugin</artifactId>
+                <version>3.4.1.1168</version>
+            </plugin>
+            <!-- end of sonar plugin -->
+            <!-- jacoco plugin -->
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <version>0.7.6.201602180812</version>
+                <!--<configuration>-->
```
Comment (joewhite101) [inline pom.xml]: Remove this if it isn't needed anymore

File: admin-console/pom.xml [hunk 2]
```diff
@@ -103,7 +103,8 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>2.12</version>
                 <configuration>
-                    <argLine>-Xmx2048m</argLine>
+                    <argLine>${surefireArgLine}</argLine>
+                    <!--<argLine>-Xmx2048m</argLine>-->
```
Comment (joewhite101) [inline admin-console/pom.xml]: Remove if not needed

File: portal-ui/pom.xml [hunk 3]
```diff
@@ -100,9 +100,10 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
-                <version>2.12</version>
+                <version>2.15</version>
                 <configuration>
-                    <argLine>-Xmx2048m</argLine>
+                    <argLine>${surefireArgLine}</argLine>
+                    <!--<argLine>-Xmx2048m</argLine>-->
```
Comment (joewhite101) [inline portal-ui/pom.xml]: Remove if not needed

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Pull request merged**